docs/configuration/config-options.md

```diff
@@ -564,6 +564,17 @@ STORAGE_AWS_REGION="us-east-2"
 STORAGE_AWS_BUCKET="my-files"
 ```
 
+### Metadata
+
+When uploading an image, Directus persists the _description, title, and tags_ from available EXIF metadata. For security
+purposes, collection of additional metadata must be configured:
+
+| Variable                   | Description                                                                                 | Default Value                                                                 |
+| -------------------------- | ------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| `FILE_METADATA_ALLOW_LIST` | A comma-separated list of metadata keys to collect during file upload. Use `*` for all[^1]. | ifd0.Make,ifd0.Model,exif.FNumber,exif.ExposureTime,exif.FocalLength,exif.ISO |
```

Comment:
Cool addition! I'd love to understand the reason behind the included metadata; is this just highly common?

Reply:
Partly because it's the metadata you'd find shown most often in other photo software, but also important here as the app tries showing any of those if it exists:

https://github.com/directus/directus/blob/9d9bcf9309023b4c0e9a56f0abd0430944b8a859/app/src/modules/files/components/file-info-sidebar-detail.vue#L83-L92

so it's partially to avoid unintended breaking changes app-side 🙂